### PR TITLE
Fix broken link to apptainer and switch udocker to link to apptainer

### PR DESCRIPTION
**Pull request by DrDaveD**
This completes #51.

### CODE REVIEW

**Comment by DrDaveD** [comment]
I am hoping that this will remove the excess "Singularity" from the list of links on the left side of the page.  Otherwise perhaps we need to remove the "title: Singularity" from content/getting-started/singularity.md.  @AkihiroSuda do you think that's needed?

**Comment by DrDaveD** [comment]
@AkihiroSuda please review.

File: content/getting-started/udocker.md [hunk 1]
```diff
@@ -19,9 +19,10 @@ udocker is _not_ [Docker](../docker).
...
 2. Fn: fakechroot mode
 3. Rn: rootless [runc](../runc) mode
-4. Sn: [Singularity](../singularity) mode
+4. Sn: [Singularity (now called Apptainer)](../apptainer) mode
 
-The third mode (Rn) falls into the scope of Rootless Containers, but the others are not:
-Pn and Fn are not containers (in our view), and Sn requires SETUID bit.
+The third mode (Rn) and fourth mode (Sn) fall into the scope of Rootless Containers,
+but the others do not:
+Pn and Fn are not containers (in our view).
```
Comment (AkihiroSuda) [inline content/getting-started/udocker.md:26]: Sn is still not a rootless container when it is configured to use setuid

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**